estate/models/estate_property.py

```diff
@@ -0,0 +1,97 @@
+from datetime import date, timedelta
+from odoo import api, models, fields, exceptions, _
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_compare
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+    _sql_constraints = [
```

Comment:
```suggestion
    _order = "id desc"

    _sql_constraints = [
```

Metadata of the model should be above constraints